common/CI/package_checks.py

```diff
@@ -304,6 +342,7 @@ def _commit_package_yaml(self, ref: str) -> Optional[Dict[str, Any]]:
 class Checker:
     checks = [
         Homepage,
+        PackageBuildDepsOrder,
```

Comment:
nit: maybe dependency orders or something similar as it checks all dep types?

Reply:
Made it `PackageDependenciesOrder`.